MAINTAINERS_GUIDE.md

```diff
@@ -63,35 +66,54 @@ by anyone is denoted by adding a comment in the pull request: `LGTM`.
-known as [Benevolent dictator for life](http://en.wikipedia.org/wiki/Benevolent_Dictator_for_Life), with Michael Crosby in the role of BDFL.
-This means that all decisions are made by default by Michael. Since making
-every decision himself would be highly un-scalable, in practice decisions
-are spread across multiple maintainers.
+Overall the maintainer system works because of mutual respect across the
+maintainers of the project.  The maintainers trust one another to make decision
+in the best interests of the project.  Sometimes maintainers can disagree and 
+this is part of a healthy project to represent the point of views of various people.
+In the case where maintainers cannot find agreement on a specific change the 
+role of a Chief Maintainer comes into play.  
 
-The relevant maintainers for a pull request can be worked out in two steps:
+The Chief Maintainer for the project is responsible for overall architecture 
+of the project to maintain conceptual integrity.  Large decisions and 
+architecture changes should be reviewed by the chief maintainer.  
+The current chief maintainer for the project is Michael Crosby (@crosbymichael).  
 
-* Step 1: Determine the subdirectories affected by the pull request. This
-  might be `netlink/` and `security/`, or any other part of the repo.
+Even though the maintainer system is built on trust, if there is a conflict
+with the chief maintainer on a decision, their decision can be challenged 
+and brought to the technical oversight board if two third of the 
+maintainers vote for an appeal. It is expected that this would be a 
+very exceptional event.
 
-* Step 2: Find the `MAINTAINERS` file which affects this directory. If the
-  directory itself does not have a `MAINTAINERS` file, work your way up
-  the repo hierarchy until you find one.
 
-### I'm a maintainer, and I'm going on holiday
+### How are maintainers added?
 
-Please let your co-maintainers and other contributors know by raising a pull
-request that comments out your `MAINTAINERS` file entry using a `#`.
+The best maintainers have a vested interest in the project.  Maintainers
+are first and foremost contributors that have shown they are committed in
+the long term success of the project.  Contributors wanting to become 
+maintainers are expected to be deeply involved contributing code, 
+pull request review, and triage of issues in the project for more than two months.
 
-### I'm a maintainer, should I make pull requests too?
+Just contributing does not make you a maintainer, it is about building trust 
+with the current maintainers of the project and being a person that they can
+depend on and trust to make decisions in the best interest of the project.  The
+final vote to add a new maintainer should be approved by over 66% of the current
+maintainers with the chief maintainer having veto power.  In case of a veto, conflict resolution rules expressed above apply.  The voting period is
```

Comment:
It seems redundant to have a two-thirds vote for approving a new maintainer, and another two-thirds vote for overriding a chief-maintainer veto.  I'd suggest either replacing:

> … with the chief maintainer having veto power. In case of a veto, conflict resolution rules expressed above apply.

with something like:

> ….  If the chief maintainer does not support a passing vote, they can appeal to the technical oversight board which can veto the approval.

Or just dropping the veto process entirely.